mitxgraders/comparers/linear_comparer.py

```diff
@@ -18,12 +18,12 @@ def get_linear_fit_error(x, y):
     Zero error in a linear relationship:
     >>> x = np.array([2, 5, 8])
     >>> result = get_linear_fit_error(x, 2*x + 1)
-    >>> round(result, 8)
+    >>> round(result, 6)
     0.0
     """
     A = np.vstack([x, np.ones(len(x))]).T
     _, residuals, _, _ = np.linalg.lstsq(A, y, rcond=-1)
-    return np.sqrt(residuals)
+    return np.sqrt(np.linalg.norm(residuals))
```

Comment:
Careful here. Are you sure you want the square root of the norm? I think that's already square rooted?

I'm guessing numpy changed what it returned for residuals?

Reply:
Good catch.

> I'm guessing numpy changed what it returned for residuals?

No. Numpy always returns an array for the residuals, but in early versions `round(np.array([ 1.23456])` would return a float, whereas in later versions it raises an error.

This caused problems for a doctest.

Besides testing issue, it bugs me that two of the `get_error...` functions return numbers, while the other two return numpy arrays (with 1 element).